handlers/main.yml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: Restart SLURM service
   service:
-    name: "{{slurm_service}}"
+    name: "{{openhpc_slurm_service}}"
```

Comment:
Note the spaces, should be "{{ openhpc_slurm_service }}", otherwise the CI job is likely to fail